### PR TITLE
Fix #2491: Reformat Code in administrator_controls_account_actions_view.xml

### LINKED ISSUE #2491: Required Reformatting for administrator_controls_account_actions_view.xml
The following files require reformatting
 - layout/administrator_controls_account_actions_view.xml
 - layout-land/administrator_controls_account_actions_view.xml
 - layout-sw600dp/administrator_controls_account_actions_view.xml 

Recommended to set up [Macros](https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros) to solve this issue

References
 - https://github.com/oppia/oppia-android/wiki/Coding-style-guide#code-formatting
 - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#macros

### PR DESCRIPTION
**Pull request by Manikant25**
## Explanation
Fixes #2491:
Reformatted code in administrator_controls_account_actions_view.xml and added Macros with shortcut macro.
  
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -5,6 +5,22 @@
...
+      <option name="PACKAGES_IMPORT_LAYOUT">
+        <value>
+          <package name="" alias="false" withSubpackages="true" />
+          <package name="java" alias="false" withSubpackages="true" />
+          <package name="javax" alias="false" withSubpackages="true" />
+          <package name="kotlin" alias="false" withSubpackages="true" />
+          <package name="" alias="true" withSubpackages="true" />
+        </value>
+      </option>
       <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="2147483647" />
```
Comment (prayutsu) [inline .idea/codeStyles/Project.xml:24]: Please revert these changes.

File: .idea/misc.xml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
```
Comment (prayutsu) [inline .idea/misc.xml:5]: Ditto

**Review by prayutsu — CHANGES_REQUESTED** [review]
Please revert the unwanted changes generated by the Android Studio

**Comment by prayutsu** [comment]
@Manikant25 Please add a colon after `Fix #2491` in the PR title and also please follow our PR guidelines as mentioned in the [wiki](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR)

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (Manikant25) [reply .idea/codeStyles/Project.xml:24]: Can you suggest how to prevent these unwanted changes done by the android studio?

Comment (rt4914) [reply .idea/codeStyles/Project.xml:24]: @Manikant25 In my experience I have seen these changes only for first couple of PRs because of your default Android Studio config. Once your PRs are merged it won't happen again.

**Review by rt4914 — COMMENTED** [review]
@Manikant25

**Comment by chrk2205** [comment]
please add FIxes:#bugnum in PR explanation
@Manikant25 
and also
to revert the changes in android studio you can use this link [here](https://stackoverflow.com/questions/38163380/how-to-revert-project-back-to-a-previous-commit-in-android-studio) and then push after comiting.

**Comment by prayutsu** [comment]
@Manikant25 Actually PR description should be something like this - 
```
Fixes #2491:
Reformatted code in administrator_controls_account_actions_view.xml
```

And please update the PR title as - 
`Fix #2491: Reformat Code in administrator_controls_account_actions_view.xml`

**Comment by prayutsu** [comment]
Also while making the new branch, give it a more meaningful name separated by hyphens (' - ') and it should consist of small case letters ideally.
A better name for this issue could be - 
`reformat-code-in-administrator_controls_account_actions_view.xml`

**Comment by Luffy18346** [comment]
@Manikant25 Any updates on this?

**Comment by Manikant25** [comment]
I am not able to understand what to push as I am not editing any code nor adding . I am justing adding macro from android studio menu

**Comment by Luffy18346** [comment]
> I am not able to understand what to push as I am not editing any code nor adding . I am justing adding macro from android studio menu

Ok If your work is completed then ask any reviewer to review this. Just type "@{reviewerName} PTAL" so that the reviewer can review this PR

**Comment by rt4914** [comment]
> I am not able to understand what to push as I am not editing any code nor adding . I am justing adding macro from android studio menu

@Manikant25 Abhay and I have made comments above. You can implement those changes and then push the code to this PR.

**Comment by Manikant25** [comment]
Thank you for guiding me, for now, I am closing this PR then push again.
